pkgs/development/python-modules/parametrize-from-file/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, coveralls
+, numpy
+, contextlib2
+, decopatch
+, more-itertools
+, nestedtext
+, pyyaml
+, tidyexc
+, toml
+}:
+
+buildPythonPackage rec {
+  pname = "parametrize-from-file";
+  version = "0.17.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "parametrize_from_file";
+    sha256 = "1c91j869n2vplvhawxc1sv8km8l53bhlxhhms43fyjsqvy351v5j";
+  };
+
+  format = "flit";
+  pythonImportsCheck = [ "parametrize_from_file" ];
+
+  checkInputs = [
+    coveralls
```

Comment:
Please patch coverage out this doesn't add value for us.

Reply:
done!